Src/VimCore/CoreInterfaces.fs

```diff
@@ -4221,7 +4221,7 @@ type IVimHost =
     abstract Make: jumpToFirstError: bool -> arguments: string -> unit
 
     /// Move the focus to the ITextView in the open document in the specified direction
-    abstract MoveFocus: textView: ITextView -> direction: Direction -> unit
+    abstract GoToWindow: textView: ITextView -> direction: WindowKind -> count: int -> unit

```

Comment:
Should `direction` be renamed to `windowKind` here?

Reply:
It should!